packages/components/src/components/hds/form/textarea/field.ts

```diff
@@ -0,0 +1,37 @@
+import templateOnlyComponent from '@ember/component/template-only';
+
+import type { HdsFormFieldSignature } from '../field';
+import type { HdsFormTextareaBaseSignature } from './base';
+import type { ComponentLike, WithBoundArgs } from '@glint/template';
+import type { HdsFormErrorSignature } from '../error';
+import type { HdsFormHelperTextSignature } from '../helper-text';
+import type { HdsFormLabelSignature } from '../label';
+import type HdsFormCharacterCountComponent from '../character-count';
+
+interface HdsFormTextareaFieldSignature {
+  Args: Omit<HdsFormFieldSignature['Args'], 'contextualClass' | 'layout'> & {
+    isInvalid?: HdsFormTextareaBaseSignature['Args']['isInvalid'];
+    value?: HdsFormTextareaBaseSignature['Args']['value'];
+    width?: string;
+    height?: string;
```

Comment:
should they come from `HdsFormTextareaBaseSignature['Args']`?

Reply:
indeed, updated in 2e40cc63eaa5c509dd66f116203756ecaab01a66